manifests/kustomize/third-party/mysql/base/mysql-deployment.yaml

```diff
@@ -20,13 +20,12 @@ spec:
       # https://dev.mysql.com/doc/refman/5.7/en/server-options.html#option_mysqld_ignore-db-dir
       # Ext4, Btrfs etc. volumes root directories have a lost+found directory that should not be treated as a database.
       - args:
-        - --ignore-db-dir=lost+found
         - --datadir
         - /var/lib/mysql
         env:
         - name: MYSQL_ALLOW_EMPTY_PASSWORD
           value: "true"
-        image: gcr.io/ml-pipeline/mysql:5.7
```

Comment:
Does MySQL 5.7 to 8 guarantee backwards compatibility? Anything affects us that changed?

Reply:
Will update with documentation on this.

Reply:
Updated in the PR description